app/controllers/slack_controller.rb

```diff
@@ -25,8 +25,20 @@ def event
     head :ok
   end
 
+  def command
```

Comment:
Of note, when we have more slash commands we can do this as a switch case, but since it's just the one now, I set it up this way.